gem_tasks/cucumber.rake

```diff
@@ -0,0 +1,27 @@
+# Copied from https://github.com/cucumber/cucumber-ruby/blob/master/gem_tasks/cucumber.rake
+
+require 'cucumber/rake/task'
+require 'cucumber/platform'
+
+class Cucumber::Rake::Task
+  def set_profile_for_current_ruby
```

Comment:
Doesn't it have support for ruby 2.3.0, the version we are using on this project? If not, do we need it?